javascript/atoms/inject.js

```diff
@@ -524,6 +524,9 @@ bot.inject.cache.getElement = function(key, opt_doc) {
     if (node == doc.documentElement) {
       return el;
     }
+    if (node.host && node.nodeType === 11) {
```

Comment:
instead of host I think that you want to `node.ownerDocument` to check if fragment is still attached.

Does this work on ShadowDOM as they are not document fragments

Reply:
I did the change.

I use that other way to try to keep the same code style. I think that it was valid (according to the last [W3C](https://www.w3.org/TR/dom41/#host)). However I think that your recommendation is faster.